hw/dv/tools/dvsim/dsim.hjson

```diff
@@ -14,7 +14,9 @@
                "+incdir+{UVM_HOME}/src",
                "{UVM_HOME}/src/uvm_pkg.sv",
                // Add dpi/vpi include path.
-               "-c-opts -I{DSIM_HOME}/include",
+               "-c-opts -I{DSIM_HOME}/../../include",
+               // Needed for including "secded_enc.h".
+               "-c-opts -I{build_dir}/src/lowrisc_dv_secded_enc_0",
```

Comment:
Sorry just noticed this as well - this is a common configuration file that needs to be free from design / testbench specific things. Please move this to an `opts.hjson` file that lives with the secded pkg.

Reply:
BTW we shouldn't need to add this. The FuseSoC generated filelist already contains `+incdir+<parent-dir-of-included-files>`. Seems like for C sources, DSIM needs an extra `-c-opts -I<path>` switch which seems superfluous. It will be great if DSIM folks can reuse the `incdir` automatically for C sources as well. 

@slaval-google

Reply:
There is no opts.hjson under hw/ip/prim/dv/prim_secded/. Did you mean where the secded sv files live? There are no opts.hjson files under hw/ip/prim...

Reply:
The latter is actually quite simple to achieve. You can add it to `dsim.hjson` itself (since it is not opentitan code specific):
Replace line 19 with:
```json
  "{eval_cmd} grep +incdir+ {sv_flist} | sed 's/+incdir+/-c-opts -I/g' | tr '\n' ' '",
```